### ISSUE TITLE
Right clicking the Terminal gives me menu in Japanese, my OS is in EN-US

### ISSUE THREAD

**Issue by FaridAhamat** [microsoft/terminal#7201]
# Environment
Windows version: Version 1903 (OS Build 18362.959)
Windows Terminal: 1.1.2021.0
Not sure how useful this information is, my Windows was installed with EN-US settings. I then installed the JP keyboard.

# Steps to reproduce
Right click on the top bar.

# Expected behavior
Show the menu in my main language setting, which is English.

# Actual behavior
The menu is in Japanese.

![JP1](https://user-images.githubusercontent.com/11820522/89540700-fc872800-d82f-11ea-935e-ba74d47fe6e8.png)

Doing the same thing on Command Prompt shows the menu in English, which is the correct behavior.

![en1](https://user-images.githubusercontent.com/11820522/89540887-39531f00-d830-11ea-89f1-a8da55d227c3.png)

**Comment by DHowett** [comment]
Can you please share a screenshot of your "preferred language" list? The order of this list dictates the language of modern packaged applications.

**Comment by DHowett** [comment]
You can drag the items in that list to rearrange them.

To quote the settings page,

_Apps and websites will appear in the first language in the list that they support._

**Comment by FaridAhamat** [comment]
Here is my Preferred languages setting:

![image](https://user-images.githubusercontent.com/11820522/89612140-a78bf600-d8b1-11ea-9fba-68f80ed8948f.png)

**Comment by FaridAhamat** [comment]
I fixed this by changing the display and preferred language to be JP. Restarted my PC. And then changing both settings back to English,

Probably the bug is on Windows, rather than Terminal.

**Comment by kreginia** [comment]
@mkurcius I'm five years late to the party, but have a look at [this comment](https://github.com/microsoft/winget-cli/issues/4056#issuecomment-2540141195) under Winget's issues for a partial fix/workaround. 
TL;DR;: Setting JP as tertiary in Windows Settings' preferred languages (so PL, EN, JP instead of PL, JP or PL, JP, EN) resolved this for me partially (when opening a new Terminal tab, "Command prompt" is still written in katakana instead of romaji).

Fact remains that this is still an issue in 2025 and that's just ridiculous.
